modules/cluster/addons/helm/aws-load-balancer-controller.yaml

```
@@ -0,0 +1,15 @@
+tolerations:
+- key: CriticalAddonsOnly
+  operator: Exists
+image:
+  repository: 602401143452.dkr.ecr.${aws_region}.amazonaws.com/amazon/aws-load-balancer-controller
+  tag: v2.1.0
```

```suggestion
  tag: v2.1.3
```

This will then match `app.kubernetes.io/version`